pages/app-layout/with-sticky-table-and-split-panel.page.tsx

```diff
@@ -60,21 +60,18 @@ const DEMO_CONTENT = (
 );
 
 export default function () {
+  const maxItemCount = 30;
   const [toolsOpen, setToolsOpen] = useState(false);
   const [selectedTool, setSelectedTool] = useState<keyof typeof toolsContent>('long');
-  const [itemCount, setItemCount] = useState<number>(30);
-  const [items, setItems] = useState(generateItems(itemCount));
+  const defaultItems = useMemo(() => generateItems(maxItemCount), []);
```

Comment:
How about putting items outside the component?
https://github.com/cloudscape-design/components/blob/main/pages/table/hooks.page.tsx#L22